code/modules/mob/mob.dm

```diff
@@ -872,7 +872,10 @@ var/list/slot_equipment_priority = list( \
 
 //this and stop_pulling really ought to be /mob/living procs
 /mob/proc/start_pulling(atom/movable/AM)
-	if( !AM || !src || src==AM || !isturf(AM.loc) )	//if there's no person pulling OR the person is pulling themself OR the object being pulled is inside something: abort!
+	if(src == AM) // Trying to pull yourself is a shortcut to stop pulling
+		stop_pulling()
+		return
+	if( !AM || !src || !isturf(AM.loc) )	//if there's no person pulling OR the object being pulled is inside something: abort!
```

Comment:
it is absolutely not fine and I will shank you if you add more at any point @taukausanake :gun::eyes: